IPython/html/static/widgets/js/widget.js

```diff
@@ -164,7 +200,7 @@ define(["widgets/js/manager",
         set_state: function (state) {
             var that = this;
             // Handle when a widget is updated via the python side.
-            return this._unpack_models(state).then(function(state) {
+            return Promise.resolve(state).then(function(state) {
```

Comment:
I created one more promise than necessary.  I could have done
```javascript
return new Promise(function(resolve, reject) {
    ....
    resolve();
})
```
but thought it would be simpler to just resolve the state to create a promise and chain a `then`.  Do you have a preference?